src/envs/satellites.py

```diff
@@ -0,0 +1,613 @@
+"""A 2D continuous satellites domain loosely inspired by the IPC domain of the
+same name.
+
+There are some number of satellites, each carrying an instrument. The possible
+instruments are: (1) a camera, (2) an infrared sensor, (3) a Geiger counter.
+Additionally, each satellite may be able to shoot Chemical X and/or Chemical
+Y. The satellites have a viewing cone within which they can see everything
+that is not occluded. The goal is to take some number of readings with
```

Comment:
can you clarify that the goal needs specific satellites to read specific objects?